src/main/java/htsjdk/tribble/AbstractFeatureReader.java

```diff
@@ -145,7 +146,7 @@ protected AbstractFeatureReader(final String path, final FeatureCodec<T, SOURCE>
     protected AbstractFeatureReader(final String path, final FeatureCodec<T, SOURCE> codec,
                                     final Function<SeekableByteChannel, SeekableByteChannel> wrapper,
                                     final Function<SeekableByteChannel, SeekableByteChannel> indexWrapper) {
-        this.path = path;
+        this.path = codec.getPathToDataFile(path);
```

Comment:
Add a comment with a one-line explanation of why we call this method here.

Reply:
Also add a comment above the declaration of the `path` instance variable clarifying that this is the path to the data file, not necessarily the file recognized by `canDecode()`

Reply:
Good idea.. Done.